src/server/app/auth/jwt/jwt-auth.strategy.ts

```diff
@@ -1,26 +1,28 @@
-import { Strategy } from 'passport-jwt';
+import { Strategy, ExtractJwt } from 'passport-jwt';
 import { PassportStrategy } from '@nestjs/passport';
 import { Injectable } from '@nestjs/common';
 import { ConfigService } from '@nestjs/config';
 import { SESSION_COOKIE_KEY } from 'src/server/config/constants';
 
 export type JwtPayload = { sub: number; username: string };
-
 @Injectable()
 export class JwtAuthStrategy extends PassportStrategy(Strategy) {
   constructor(configService: ConfigService) {
-    const extractJwtFromCookie = (req) => {
+    const extractJwtFromRequest = (req) => {
       let token = null;
 
-      if (req && req.cookies) {
+      if (ExtractJwt.fromAuthHeaderAsBearerToken()(req)) {
+        token = ExtractJwt.fromAuthHeaderAsBearerToken()(req);
```

Comment:
As you [mentioned](https://github.com/thisismydesign/nestjs-starter/issues/52#issuecomment-1178499583), this was removed in https://github.com/thisismydesign/nestjs-starter/pull/29. Can you explain why this is needed? I will try the GQL playground later but afaict the cookie is set upon login and then read here, should work out of the box.